skimage/transform/histogram_matching.py

```diff
@@ -48,12 +53,10 @@ def match_histograms(image, reference, multichannel=False):
 
     References
     ----------
-    .. [1] http://paulbourke.net/miscellaneous/equalisation/
+     .. [1] http://paulbourke.net/miscellaneous/equalisation/
+     .. [2] https://www.researchgate.net/post/Histogram_matching_for_color_images
```

Comment:
```suggestion
    .. [2] https://www.researchgate.net/post/Histogram_matching_for_color_images
```

Reply:
I accepted your earlier suggestion and did not do any change in this line, so it seems you are correcting your earlier suggestion... :]